tools/model-compiler/build.gradle

```diff
@@ -31,42 +31,40 @@ ext {
 
 group = 'io.spine.tools'
 
-apply from: deps.scripts.generateDescriptorSet
-
 dependencies {
-    compile project(':plugin-base')
-    compile project(':base')
-    compile project(':codegen')
+    implementation project(':plugin-base')
+    implementation project(':base')
 
     implementation (group: 'com.google.template', name: 'soy', version: '2017-02-01') {
         exclude group: 'com.google.guava'
     }
 
     implementation deps.build.protobuf
     implementation deps.gen.javaPoet
-    
+
     // A library for parsing Java sources.
     // Used for parsing Java sources generated from Protobuf files
     // to make their annotation more convenient.
-    compile (group: 'org.jboss.forge.roaster', name: 'roaster-api', version: roasterVersion) {
+    implementation (group: 'org.jboss.forge.roaster', name: 'roaster-api', version: roasterVersion) {
```

Comment:
As long as you do this, you may do the `compile` -> `implementation` / `api` migration in the rest of modules.